.drone/drone.yml

```diff
@@ -1203,19 +1203,19 @@ steps:
 - commands:
   - apk add --no-cache bash git
   - git fetch origin --tags
-  - echo $(./tools/image-tag)
   - echo $(./tools/image-tag) > .tag
-  - export RELEASE_NAME=$([[ $DRONE_SOURCE_BRANCH =~ $RELEASE_BRANCH_REGEXP ]] &&
-    echo $DRONE_SOURCE_BRANCH | grep -oE "([0-9\.x]+)" | sed "s/\./-/g" || echo "next")
+  - export RELEASE_TAG=$(cat .tag)
+  - export RELEASE_NAME=$([[ $RELEASE_TAG =~ $RELEASE_TAG_REGEXP ]] && echo $RELEASE_TAG
+    | grep -oE "([0-9]+\.[0-9]+)" | sed "s/\./-/g" | sed "s/$/-x/" || echo "next")
```

Comment:
Can we give some name to `([0-9]+\.[0-9]+`? similar to `RELEASE_TAG_REGEXP`?

Q: is it different than the one used above? (`([0-9]+\\.[0-9]+)`) only difference I see is `backslash` escaped in later (`\\.`). Is this intentional?

Reply:
in jsonnet we used `\\.` because we need to generate value `\.` in yaml file. if we do not escape `\` in jsonnet, it generates just `.` in yaml file )))